governance/handler.go

```diff
@@ -127,22 +156,45 @@ func (g *Governance) AddVote(key string, val interface{}) bool {
 
 func (g *Governance) adjustValueType(key string, val interface{}) interface{} {
 	k := GovernanceKeyMap[key]
-	reqType := GovernanceItems[k].t
 	var x interface{}
 
 	// When an int value comes from JS console, it comes as a float64
-	if reqType == uint64T && reflect.TypeOf(val) == float64T {
+	if GovernanceItems[k].checkValueType(uint64(0)) && reflect.TypeOf(val) == float64T {
 		x = uint64(val.(float64))
 		if float64(x.(uint64)) == val.(float64) {
 			return x
 		}
 		return val
 	}
 
-	// address comes as a form of string from JS console
-	if reqType == addressT && reflect.TypeOf(val) == stringT {
-		if common.IsHexAddress(val.(string)) {
-			x = common.HexToAddress(val.(string))
+	if GovernanceItems[k].checkValueType(common.Address{}) && reflect.TypeOf(val) == stringT {
+		addresses := strings.Split(val.(string), ",")
```

Comment:
This is not a good practice. If val.(string) fails, the process can panic. This cannot be true because there is a condition statement of type checking, but it would be better to use like below:

```go
addressesStr, ok := val.(string)
if !ok {
  // do something for a failure case
}
// do something for a successful case
```

Reply:
I refactored whole `adjustValueType` at https://github.com/klaytn/klaytn/pull/1110/commits/70ad97142c82c7962c5150839e6fb074ae5091f3. Please take another look.